bucket/cmder.json

```diff
@@ -3,16 +3,22 @@
     "description": "Portable console emulator for Windows",
     "version": "1.3.11",
     "license": "MIT",
-    "persist": "config",
     "url": "https://github.com/cmderdev/cmder/releases/download/v1.3.11/cmder_mini.zip",
     "hash": "23c938b2495ef620de816747dfb2c0d9ad7884aec1d9923dbb7a94e6c91df849",
+    "installer": {
+        "script": "if (!(Test-Path \"$dir\\vendor\\conemu-maximus5\\ConEmu.xml\")) {Set-Content \"$dir\\vendor\\conemu-maximus5\\ConEmu.xml\" $Null}"
```

Comment:
- I do not see a reason to use installer as pre_install block is more ergonomic
- No need to set `$null`. Also it is `$null` not `$Null`
    - There is default, which could be copied.

If you are setting $null, then use new-item to save file opening and opening buffer

https://i.imgur.com/KCDsq9D.png